APIBlueprintImporter.coffee

```diff
@@ -199,8 +199,10 @@ APIBlueprintImporter = ->
       blueprint = JSON.parse(http_request.responseBody)
       @importBlueprint context, blueprint
       return true
-
-    throw new Error "HTTP Request failed: " + http_request.responseStatusCode
+    if (http_request.responseStatusCode is 422)
+      throw new Error "Parsing failed. Check your blueprint for either warning or errors."
```

Comment:
Yes, it would be great if you can check the `errors` component of the API result. If there is any errors, we could perhaps show the user the error. If there are warnings we should be able to continue.

There was a breaking change in the API where parsing a blueprints with a warning now returns a 422. Previously it returned a 200, and only a 400 on an error.